app/src/main/java/org/stepic/droid/ui/adapters/TagsAdapter.kt

```diff
@@ -0,0 +1,63 @@
+package org.stepic.droid.ui.adapters
+
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import kotlinx.android.synthetic.main.tag_item.view.*
+import org.stepic.droid.R
+import org.stepic.droid.model.Tag
+import org.stepic.droid.ui.listeners.OnItemClickListener
+
+class TagsAdapter(private val onTagClicked: (Tag) -> Unit) : RecyclerView.Adapter<TagsAdapter.TagViewHolder>() {
+
+    private val _tags: MutableList<Tag> = mutableListOf()
+
+    fun setTags(tags: List<Tag>) {
```

Comment:
Can be written as custom setter for `_tags` field.

Reply:
It cannot, because `_tags` is immutable